client/src/components/manageProjects/editMeetingTimes.js

```diff
@@ -41,13 +41,10 @@ const EditMeetingTimes = ({
         };
       }
 
-      //if there is a description or a blank description
-      if (values.description || !values.description) {
```

Comment:
This portion of the code always evaluates to `true`, remove if block

Reply:
I feel like this should be 

if (values.description) {